lnbits/extensions/bleskomat/exchange_rates.py

```diff
@@ -80,7 +80,5 @@ async def fetch_fiat_exchange_rate(currency: str, provider: str):
     else:
         data = {}
     getter = exchange_rate_providers[provider]["getter"]
-    print(getter)
-    if callable(getter):
-        rate = float(getter(data, replacements))
-    return rate
+    assert callable(getter), "cannot call getter function"
```

Comment:
- this replaces a `return None` with an `AssertionFailure`
- should be tested if it behaves correctly

Reply:
e0d9f2d7